beetsplug/lastgenre/genres-tree.yaml

```diff
@@ -156,6 +156,31 @@
     - timba
     - twoubadou
     - zouk
+- classical:
+    - ballet
+    - baroque:
+        - baroque music
+    - cantata
+    - chamber music:
+        - string quartet
+    - classic
```

Comment:
I've not imported my classical CD's yet, but when I do it would be great to have a Classical genre, and not classic - as you point out, that means something different and is far too vague.